paper-tristate-checkbox.html

```diff
@@ -311,40 +305,49 @@
           type: String,
           value: null,
           reflectToAttribute: true,
-          notify: true
+          notify: true,
         },
 
         /**
          * The element's value when `'on'`/checked.
          */
         checked: {
           type: String,
-          value: "x"
+          value: 'x',
         },
 
         /**
          * The element's value when `'off'`/unchecked.
          */
         unchecked: {
           type: String,
-          value: " "
+          value: ' ',
         },
 
         /**
          * The element's value when `null`/indeterminate.
          */
         indeterminate: {
           type: String,
-          value: ""
+          value: '',
         },
 
         /**
          * The element's value based on its current state.
          */
         value: {
           type: String,
-          computed: '_computeValue(state)'
-        }
+          computed: '_computeValue(state)',
+        },
+      },
+
+      attached() {
+        this.setAttribute('aria-checked', 'mixed');
+        this._computeValue(this.state);
+        this.addEventListener('click', this._regularTap);
```

Comment:
You need a ready block to bind the function:
`this._regularTap = this._regularTap.bind(this);`